server/src/main/java/org/elasticsearch/search/aggregations/support/TimeSeriesIndexSearcher.java

```diff
@@ -76,6 +78,10 @@ public TimeSeriesIndexSearcher(IndexSearcher searcher, List<Runnable> cancellati
         this.timestampReverse = TIME_SERIES_SORT[1].getOrder() == SortOrder.DESC;
     }
 
+    public void setMinimumScore(float minimumScore) {
```

Comment:
This does need to be invoked from `AggregationPhase` otherwise it minimum score functionality can't be used?